src/Utils.purs

```diff
@@ -36,21 +36,39 @@ inContainer :: ∀ item t f. (Unit -> t Unit) -> f -> Container.Query item t f
 inContainer = Container.ParentQuery <<< H.action
 
 getInputProps = augmentHTML
-  [ HE.onFocus      $ HE.input_ $ Container.Visibility Container.Toggle
-  , HE.onKeyDown    $ HE.input  $ Container.Key
+  [ HE.onFocus     $ HE.input_ $ Container.Visibility Container.Toggle
+  , HE.onKeyDown   $ HE.input  $ Container.Key
+  , HE.onMouseDown $ HE.input_ $ Container.Mouse Container.Down
+  , HE.onMouseUp   $ HE.input_ $ Container.Mouse Container.Up
+  , HE.onBlur      $ HE.input_ $ Container.Blur
   , HP.tabIndex 0
   ]
 
 getToggleProps = augmentHTML
   [ HE.onClick     $ HE.input_ $ Container.Visibility Container.Toggle
   , HE.onKeyDown   $ HE.input  $ Container.Key
--- , HE.onBlur      $ HE.input_ $ Container.Visibility Menu.Off  -- can be re-added if wanted
+  , HE.onMouseDown $ HE.input_ $ Container.Mouse Container.Down
+  , HE.onMouseUp   $ HE.input_ $ Container.Mouse Container.Up
+  , HE.onBlur      $ HE.input_ $ Container.Blur
+  , HP.tabIndex 0
+  ]
+
+getContainerProps = augmentHTML
+  [ HE.onMouseDown $ HE.input_ $ Container.Mouse Container.Down
+  , HE.onMouseUp   $ HE.input_ $ Container.Mouse Container.Up
+  , HE.onBlur      $ HE.input_ $ Container.Blur
+  , HP.tabIndex 0
+  ]
+
+getChildProps = augmentHTML
```

Comment:
@whoadave Is this something that we would provide to end users of the lib? Or is this an example of something they'd write custom to add props to something they're putting into the render function?

Reply:
This is something we provide to users of the lib. It needs to be used for any elements rendered inside the container that can steal away focus from the toggle or item elements e.g. custom buttons, links, form controls in a dropdown

If you build the example and run it, you can see how it's applied to the "Click Me" button. Without the childProps, clicking on the button and then clicking away would not hide the dropdown as expected